coursedata/texts/en.yaml

```diff
@@ -56,6 +57,32 @@ ClientErrorMessages:
     CheckInternet: "Have a look if your Internet connection is working properly?"
     ServerError: "You wrote a program we weren't expecting. If you want to help, send us an email with the level and your program at hedy@felienne.com. In the mean time, try something a little different and take another look at the examples. Thanks!"
     Program_too_long: "Your program takes too long to run."
+    New_subjects:
```

Comment:
Here too can we use new_commands, and... I know it is more complex but can we extract this from hedy.py?

We have a list of commands per level there too and it would be nicer if we do not have to do double bookkeeping

Reply:
If you convert the lists into sets you can do a set subtraction to get the difference I think, something like `set(list_2) - set(list_1)`

Reply:
We thought it would be best to have a different set of commands per level. Because in later levels there are no new commands learned, only the way the commands work has changed.  In level 4 the ' character is introduced and these are not a command.

Reply:
Ok, I can see how we prefer to keep this list, but can we rename it to `new_commands`? Subjects is a term that we do not use in other places so that will be confusing.